example_script/check_wiki_links_studies.R

```diff
@@ -0,0 +1,26 @@
+## load necessary packages
+
+library(reticulate)
+
+use_condaenv("r-reticulate", required = T) 
+
+py_module_available("synapseclient") 
+py_module_available("pandas")
+
+library(nfportalutils)
+library(dplyr)
+
+## login, assuming creds are set with the following env vars
+nfportalutils::syn_login(Sys.getenv("SYNAPSE_USER"), Sys.getenv("SYNAPSE_PWD"))   
```

Comment:
```suggestion
nfportalutils::syn_login(Sys.getenv("SYNAPSE_AUTH_TOKEN"))   
```

Suggest changing this to PAT as that seems to be preferred over username/pwd (and no need for username this way).

Reply:
Yeah, we need to update the wrapper for login to allow this. Currently, this won't actually work to log in because username and password are expected: https://github.com/nf-osi/nfportalutils/blob/develop/R/login.R#L12-L15. I will hopefully update `syn_login` and examples using `syn_login` in future PR.

Reply:
Oops, good point :) I kind of forgot about the wrapper function.